### PR TITLE
Creating simplecove.js

### PR DESCRIPTION
**Pull request by SimpleCove**
Adding simplecove functionality to RES

### CODE REVIEW

File: lib/modules/hosts/simplecove.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	async handleLink(elem, [, id]) {
+		const data = await ajax({
+			url: `http://www.simplecove.com/resapi/${id}`,
+			type: 'json',
+		});
+
+		const images = data.map(x =>
+			({
+				src: x.photo_src.replace('http:', 'https:'),
+				caption: x.photo_caption.replace(/\/?p/g, ''),
```
Comment (erikdesjardins) [inline lib/modules/hosts/simplecove.js]: `.replace(/\/?p/g, '')` isn't necessary after #3049

Comment (SimpleCove) [inline lib/modules/hosts/simplecove.js]: I've removed it.

Thank you

File: lib/modules/hosts/simplecove.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { ajax } from '../../environment';
+
+export default {
+	moduleID: 'simplecove',
+	name: 'SimpleCove',
+	domains: ['simplecove.com'],
+	logo: 'https://simplecove.com/static/images/reslogo.jpg',
+	detect: ({ href }) => (/^https?:\/\/(?:www\.)?simplecove\.com\/(\d+)(?:\/)?.+/g).exec(href),
```
Comment (erikdesjardins) [inline lib/modules/hosts/simplecove.js]: `.+` at the end is redundant, this regex is effectively equivalent: `^https?:\/\/(?:www\.)?simplecove\.com\/(\d+).`

You probably want `^https?:\/\/(?:www\.)?simplecove\.com\/(\d+)(?:\/|$)` or just `^https?:\/\/(?:www\.)?simplecove\.com\/(\d+)`

Comment (SimpleCove) [inline lib/modules/hosts/simplecove.js]: You are correct. I changed it to ^https?:\/\/(?:www\.)?simplecove\.com\/(\d+) . Thank you.

File: lib/modules/hosts/simplecove.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import { ajax } from '../../environment';
+
+export default {
+	moduleID: 'simplecove',
+	name: 'SimpleCove',
+	domains: ['simplecove.com'],
+	logo: 'https://simplecove.com/static/images/reslogo.jpg',
+	detect: ({ href }) => (/^https?:\/\/(?:www\.)?simplecove\.com\/(\d+)).exec(href),
```
Comment (erikdesjardins) [inline lib/modules/hosts/simplecove.js]: Missing the end of the regex; should be: `(/^https?:\/\/(?:www\.)?simplecove\.com\/(\d+)/i).exec(href),`

Comment (SimpleCove) [inline lib/modules/hosts/simplecove.js]: Thank you for the tip. I've fixed it.

**Comment by erikdesjardins** [comment]
Looks good, thanks!

**Pull request merged**